### DISCUSSION TITLE
reading sprint

### DISCUSSION THREAD

**Discussion by noraspellman** [pengowray/sprinto#8] [Ideas]
I've been working on this idea for a while, and I have no knowledge of how to do it myself, (I've tried). I would love to see a reading sprint bot instead of writing, so instead of words, it could be pages. Currently, my book-club server uses sprinto and treats it as a reading bot, but id love to actually have a proper reading bot!

**Comment by HeyMrsMay** [comment]
I agree this would be super helpful in book club discords!

**Comment by KoreUrala** [comment]
I'd like to be able to use the bot for drawing.
just a mode without word count functionality and maybe able to get a random reference to draw
